lib/plugins/aws/deploy/lib/core-cf.json

```diff
@@ -30,7 +30,7 @@
     "IamPolicyLambda": {
       "Type": "AWS::IAM::Policy",
       "Properties": {
-        "PolicyName": "${stage}-${project}-lambda",
+        "PolicyName": "",
```

Comment:
it's an outdated string from v0 that gets replaced anyway in code. [Check this line](https://github.com/serverless/serverless/blob/master/lib/plugins/aws/deploy/lib/initializeResources.js#L24)